packages/synapse-react-client/src/synapse-queries/dataaccess/useAccessRequirements.ts

```diff
@@ -158,6 +158,7 @@ export function useCreateLockAccessRequirement(
       if (options?.onSuccess) {
         return options.onSuccess(data, variables, ctx)
       }
+      return
```

Comment:
These functions must have a return statement. If I had to guess, it probably has something to do with [this TypeScript change](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-1.html) and some (outdated) react-query types.